misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/DockerDynamoDbModule.kt

```diff
@@ -24,6 +24,12 @@ class DockerDynamoDbModule(
   fun providesAmazonDynamoDB(): AmazonDynamoDB {
     return DockerDynamoDb.connect()
   }
+
+  companion object {
+    fun withTables(vararg tables: KClass<*>): DockerDynamoDbModule {
```

Comment:
I don’t like how this API is asymmetric, with the List variant in the constructor and the varargs one as a factory method.

Maybe make this a constructor overload instead? Or hide the constructor and add a factory method?